### PR TITLE
Singleton lazy loaded property getter

### PR DESCRIPTION
**Pull request by moritz-t-w**
The method `getInstance` of `Singleton` is an implicit getter. So I upgraded it to an actual getter.

### CODE REVIEW

File: src/Singleton/Conceptual/index.ts [hunk 1]
```diff
@@ -11,10 +11,10 @@
  */
 
 /**
- * EN: The Singleton class defines the `getInstance` method that lets clients
+ * EN: The Singleton class defines an `instance` getter, that lets clients
  * access the unique singleton instance.
  *
- * RU: Класс Одиночка предоставляет метод getInstance, который позволяет
+ * RU: Класс Одиночка определяет геттер `instance`, который позволяет
```
Comment (moritz-t-w) [inline src/Singleton/Conceptual/index.ts:17]: I used ChatGPT to generate this, as explained in the commit message.

File: src/Singleton/Conceptual/index.ts [hunk 2]
```diff
@@ -30,17 +30,17 @@ class Singleton {
...
 
     /**
-     * EN: The static method that controls the access to the singleton instance.
+     * EN: The static getter that controls access to the singleton instance.
      *
      * This implementation let you subclass the Singleton class while keeping
      * just one instance of each subclass around.
      *
-     * RU: Статический метод, управляющий доступом к экземпляру одиночки.
+     * RU: Статический геттер, управляющий доступом к экземпляру одиночки.
```
Comment (moritz-t-w) [inline src/Singleton/Conceptual/index.ts:38]: I used ChatGPT to generate this, as explained in the commit message.

**Review by moritz-t-w — COMMENTED** [review]
Maybe someone wants to check the russian

**Comment by neochief** [comment]
Thanks!

**Pull request merged**